scripts/lib/cluster.sh

```diff
@@ -18,17 +18,17 @@ function up-test-cluster() {
     AWS_K8S_TESTER_EKS_NAME=$CLUSTER_NAME \
         AWS_K8S_TESTER_EKS_KUBECONFIG_PATH=$KUBECONFIG_PATH \
         AWS_K8S_TESTER_EKS_PARAMETERS_VERSION=${K8S_VERSION%.*} \
+        AWS_K8S_TESTER_EKS_PARAMETERS_ENCRYPTION_CMK_CREATE=false \
+        AWS_K8S_TESTER_EKS_PARAMETERS_ROLE_CREATE=false \
+        AWS_K8S_TESTER_EKS_PARAMETERS_ROLE_ARN=arn:aws:iam::404174646922:role/K8sTester-ClusterManagementRole \
         AWS_K8S_TESTER_EKS_ADD_ON_MANAGED_NODE_GROUPS_ENABLE=true \
+        AWS_K8S_TESTER_EKS_ADD_ON_MANAGED_NODE_GROUPS_ROLE_CREATE=false \
+        AWS_K8S_TESTER_EKS_ADD_ON_MANAGED_NODE_GROUPS_ROLE_ARN=arn:aws:iam::404174646922:role/K8sTester-ClusterManagementRole \
         AWS_K8S_TESTER_EKS_ADD_ON_MANAGED_NODE_GROUPS_MNGS={\"${CLUSTER_NAME}-mng-for-cni\":{\"name\":\"${CLUSTER_NAME}-mng-for-cni\",\"tags\":{\"group\":\"amazon-vpc-cni-k8s\"},\"ami-type\":\"AL2_x86_64\",\"asg-min-size\":3,\"asg-max-size\":3,\"asg-desired-capacity\":3,\"instance-types\":[\"c5.xlarge\"]}} \
         AWS_K8S_TESTER_EKS_ADD_ON_MANAGED_NODE_GROUPS_REMOTE_ACCESS_PRIVATE_KEY_PATH=$SSH_KEY_PATH \
         AWS_K8S_TESTER_EKS_AWS_IAM_AUTHENTICATOR_PATH=$AUTHENTICATOR_PATH \
-        AWS_K8S_TESTER_EKS_ADD_ON_JOB_ECHO_ENABLE=true \
-        AWS_K8S_TESTER_EKS_ADD_ON_JOB_ECHO_PARALLELS=3 \
-        AWS_K8S_TESTER_EKS_ADD_ON_JOB_ECHO_COMPLETES=30 \
-        AWS_K8S_TESTER_EKS_ADD_ON_JOB_PERL_ENABLE=true \
```

Comment:
The above were added (by me) to prevent the default 1000-Job echo Pod test from occurring. Any reason the above were deleted, @SaranBalaji90 and @gyuho?

Reply:
I see now that https://github.com/aws/aws-k8s-tester/commit/0210a3524c25f2954ae4028bf785fcdc29446421 changed the default to not enabled for the echo, perl and secrets jobs. Tough to tell, since the commit message for the above commit just says "eksconfig: update" :(